From 659125618aeeeea91f322ba74973398c979053aa Mon Sep 17 00:00:00 2001
From: Chirayu Desai <chirayudesai1@gmail.com>
Date: Thu, 27 Nov 2014 21:00:01 +0530
Subject: [PATCH 04/12] SELinux: su: Remove extra quote in a comment

* Fixes
  vendor/cm/sepolicy/su.te:46:WARNING 'unrecognized character' at token '''

Change-Id: I3957ba7ac05062766cbf6c8f3c3975f20c95532e
---
 common/public/su.te | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/common/public/su.te b/common/public/su.te
index 045de39..76e4176 100644
--- a/common/public/su.te
+++ b/common/public/su.te
@@ -54,7 +54,7 @@ userdebug_or_eng(`
   allow suclient superuser_device:dir { create rw_dir_perms setattr unlink };
   allow suclient superuser_device:sock_file { create setattr unlink write };
   allow suclient untrusted_app_devpts:chr_file { read write ioctl };
-  # For Settings' control of access
+  # For Settings control of access
   allow system_app superuser_device:sock_file { read write create setattr unlink getattr };
   allow system_app sudaemon:unix_stream_socket { connectto read write setopt ioctl };
   allow system_app superuser_device:dir { create rw_dir_perms setattr unlink };
-- 
2.7.4

